vaadin-development-mode-detector.html

```diff
@@ -15,7 +17,7 @@
 
   function isDevelopmentMode() {
     try {
-      return isForcedDevelopmentMode() || (isLocalhost() && !isMinified() && !isFlowProductionMode());
+      return isForcedDevelopmentMode() || (FlowClients && !isFlowProductionMode()) || (isLocalhost() && !isMinified());
```

Comment:
This seems to make Flow be in development mode even when not on localhost. Maybe it would be clearer to make it something like

```
if (isForceDevelopmentMode()) {
   return true;

if (!isLocalhost()) {
  return false;
}

if (FlowClients) {
  return !isFlowProductionMode();
}

return !isMinified();
```

Reply:
Done.